Zend/tests/bug42859.phpt

```diff
@@ -9,4 +9,4 @@ use Blah\Exception;
 use Blah\Ex;
 ?>
 --EXPECTF--
-Fatal error: Cannot use Blah\Ex as Ex because the name is already in use in %sbug42859.php on line 6
+Fatal error: Cannot import class Blah\Ex as Ex, Ex has already been declared in %s on line %d
```

Comment:
So I normalized the different error messages related to the use statement to use `import`. i think this change doesn't make them significantly less clear, but I felt that they are a bit more "smooth". So all in all, I don't mind whichever expression we end up with.